content/applications/finance/fiscal_localizations/peru.rst

```diff
@@ -731,7 +731,7 @@ Next, you need to retrieve the *client ID* and *client secret* from |SUNAT|. To
    differ from the user set for electronic invoicing.
 
 These credentials should be used to configure the delivery guide general settings from
-:menuselection:`Accounting --> Configuration --> Settings --> Peruvian Electronic Invoicing`.
+:menuselection:`Inventory --> Configuration --> Settings --> Peru Delivery Guide`.
```

Comment:
Hi small change to be more descriptive and rely less on the image for context. I also broke the "Peru Delivery Guide" section out of the menuselection per [our guidelines](https://www.odoo.com/documentation/master/contributing/documentation/rst_guidelines.html#menu-selection).
```suggestion
:menuselection:`Inventory --> Configuration --> Settings`, and scroll down to the :guilabel:`Peru
Delivery Guide` section.

Configure the following :guilabel:`Sunat Delivery Guide API` fields:

- :guilabel:`Guide Client ID`
- :guilabel:`Guide Client Secret`
- :guilabel:`Guide SOL User`
- :guilabel:`Guide SOL Password`
```

Reply:
Beautiful! thanks @samueljlieber